stdlib/io/src/main/ballerina/src/io/writable_data_channel.bal

```diff
@@ -42,74 +42,101 @@ public type WritableDataChannel object {
+# ```ballerina
+# io:Error? err = dataChannel.writeInt32(length);
+# ```
+# 
+# + value - integer which will be written
+# + return - nill if the content is written successfully, or else `Error` if any error occurred
     public function writeInt32(int value) returns Error? {
         return writeInt32Extern(self, value);
     }
 
-    # Writes 64 bit integer.
-    #
-    # + value - integer which will be written
-    # + return - nill if the content is written successfully or `Error` if any error occurred
+# Writes 64 bit integer.
+# ```ballerina
+# io:Error? err = dataChannel.writeInt64(length);
+# ```
+#
+# + value - integer which will be written
+# + return - nill if the content is written successfully, or else `Error` if any error occurred
     public function writeInt64(int value) returns Error? {
         return writeInt64Extern(self, value);
     }
 
-    # Writes 32 bit float.
-    #
-    # + value - float which will be written
-    # + return - nill if the float is written successfully or `Error` if any error occurred
+# Writes 32 bit float.
+# ```ballerina
+# io:Error? err = dataChannel.writeFloat32(3.12);
+# ```
+#
+# + value - float which will be written
+# + return - nill if the float is written successfully, or else `Error` if any error occurred
     public function writeFloat32(float value) returns Error? {
         return writeFloat32Extern(self, value);
     }
 
-    # Writes 64 bit float.
-    #
-    # + value - float which will be written
-    # + return - nill if the float is written successfully or `Error` if any error occurred
+# Writes 64 bit float.
+# ```ballerina
+# io:Error? err = dataChannel.writeFloat32(3.12);
+# ```
+#
+# + value - float which will be written
+# + return - nill if the float is written successfully, or else `Error` if any error occurred
     public function writeFloat64(float value) returns Error? {
         return writeFloat64Extern(self, value);
     }
 
-    # Writes boolean.
-    #
-    # + value - boolean which will be written
-    # + return - nill if the content is written successfully or `Error` if any error occurred
+# Writes boolean.
+# ```ballerina
+# io:Error? err = dataChannel.writeInt64(length);
+# ```
+#
+# + value - boolean which will be written
+# + return - nill if the content is written successfully, or else `Error` if any error occurred
     public function writeBool(boolean value) returns Error? {
         return writeBoolExtern(self, value);
     }
 
-    # Writes a given string value to the respective channel.
-    #
-    # + value - the value which should be written
-    # + encoding - the encoding which will represent the value string
-    # + return - nill if the content is written successfully or `Error` if any error occurred
+# Writes a given string value to the respective channel.
+# ```ballerina
+# io:Error? err = dataChannel.writeString(record);
+# ```
+#
+# + value - the value which should be written
+# + encoding - the encoding which will represent the value string
```

Comment:
```suggestion
# + encoding - The encoding, which will represent the value string
```
Please check if all parameter descriptions start with a capital letter in all occurrences.